packages/fx-core/src/plugins/resource/frontend/constants.ts

```diff
@@ -50,6 +50,11 @@ export class EnvironmentVariables {
 export class FrontendPathInfo {
   static WorkingDir = "tabs";
   static TemplateDir = path.join("plugins", "resource", "frontend");
+  static bicepTemplateFolderName = "bicep";
```

Comment:
@hund030  What's our code style of static variable of Class? I see some are camelCase, some are PascalCase

Reply:
Let me create a PR to make it consistent.